pkg/reconciler/names.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2018 The Knative Authors
```

Comment:
<!--copyright-bot-->
```suggestion
Copyright 2019 The Knative Authors
```

Reply:
Hmm, IDK how I managed to bring this back...  Bad rebase, I suppose...